ti/instrumentation/python/pytest.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2021 Harness Inc. All rights reserved.
+// Use of this source code is governed by the PolyForm Free Trial 1.0.0 license
+// that can be found in the licenses directory at the root of this repository, also available at
+// https://polyformproject.org/wp-content/uploads/2020/05/PolyForm-Free-Trial-1.0.0.txt.
+
+package python
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/harness/lite-engine/internal/filesystem"
+	"github.com/harness/lite-engine/ti"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	pytestCmd = "pytest"
+)
+
+type pytestRunner struct {
+	fs  filesystem.FileSystem
+	log *logrus.Logger
+}
+
+func NewPytestRunner(log *logrus.Logger, fs filesystem.FileSystem) *pytestRunner { //nolint:revive
+	log.Infoln("NewPytestRunner starting")
+
+	return &pytestRunner{
+		fs:  fs,
+		log: log,
+	}
+}
+
+func (m *pytestRunner) AutoDetectPackages(workspace string) ([]string, error) {
+	return []string{}, errors.New("not implemented")
+}
+
+func (m *pytestRunner) AutoDetectTests(ctx context.Context, workspace string, testGlobs []string) ([]ti.RunnableTest, error) {
+	tests := make([]ti.RunnableTest, 0)
+	pythonTests, err := GetPythonTests(testGlobs)
+	if err != nil {
+		return tests, err
+	}
+	tests = append(tests, pythonTests...)
+	return tests, nil
+}
+
+func (m *pytestRunner) GetCmd(ctx context.Context, tests []ti.RunnableTest, userArgs, workspace,
+	agentConfigPath, agentInstallDir string, ignoreInstr, runAll bool) (string, error) {
+	// Run all the tests
+	m.log.Infoln("Pytest GetCmd starting")
+
+	if runAll {
+		if ignoreInstr {
+			return strings.TrimSpace(fmt.Sprintf("%s %s", pytestCmd, userArgs)), nil
+		}
+		return strings.TrimSpace(
```

Comment:
Define `"."` as constant and use it here

Reply:
Can you also look if it's possible to build the `agentArg` before hand and just add it here? For example: https://github.com/harness/lite-engine/blob/6d53379909358cdf3c9179e8ab64af5560771990/ti/instrumentation/java/maven.go#L57